components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -88,6 +88,8 @@ alias grep='grep --color=auto'
 alias egrep='egrep --color=auto'
 alias fgrep='fgrep --color=auto'
 
+# Set TERMINFO so hab can give us a delightful experience.
+export TERMINFO=$(_pkgpath_for core/ncurses)/share/terminfo
 PROFILE
```

Comment:
Could you add one empty line between 92 and 93. Looks like this file gets appended to after and runs a comment block into it. Not a big deal, but will make `/etc/profile` look stellar!